torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -565,13 +602,33 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
         nullptr));
   }
 
-  if (measure_kernel_time_) {
+  if (measure_kernel_time_ ||
+      isDebugDumpEnabled(DebugDumpOption::EffectiveBandwidth)) {
     cudaEventRecord(finish_event);
     cudaEventSynchronize(start_event);
     cudaEventSynchronize(finish_event);
     cudaEventElapsedTime(&kernel_time_ms_, start_event, finish_event);
     cudaEventDestroy(start_event);
     cudaEventDestroy(finish_event);
+
+    if (isDebugDumpEnabled(DebugDumpOption::EffectiveBandwidth)) {
```

Comment:
Ins't this conditional redundant as you already get into this set of code to measure the bandwidth at lines 605-606?